index.html

```diff
@@ -0,0 +1,265 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;600;700&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+  <title>Sir Malcom C. Maina</title>
+</head>
+<body>
+  <section id="home">
+    <header class="navbar">
+      <div class="logo-section">
+        <a href="#home">Sir Malcom</a>
+      </div>
+      <nav class="links-section">
+        <a href="#portfolio" class="lnks">Portfolio</a>
+        <a href="#about" class="lnks">About</a>
+        <a href="#contact" class="lnks">Contact</a>
+      </nav>
+    </header>
```

Comment:
- [ ] On the desktop view, the padding of this section of the `header` which contains the `nav` items seems to be a little enclosed. In reference to the figma design, kindly adjust it in comparison to the pictures I attached. This is because a typical developer job involves meeting clients' needs as close as possible if not to perfection 😉 . But the goal is to get you to replicate designs as given. 

# Your Design:
![image](https://user-images.githubusercontent.com/50990925/213592289-ef1e814b-7d3f-4a53-bea9-ca4c1acf3611.png)

# Figma:
![image](https://user-images.githubusercontent.com/50990925/213592350-b2f9f2f9-a91d-4b8a-b2f9-f4751f0ec2d3.png)